com.unity.render-pipelines.universal/Shaders/LitInput.hlsl

```diff
@@ -15,10 +15,8 @@ half _Smoothness;
 half _Metallic;
 half _BumpScale;
 half _OcclusionStrength;
-#if defined(_CLEARCOAT) || defined(_CLEARCOATMAP)
 half _ClearCoatMask;
-//half _ClearCoatSmoothness; // TODO: enable
-#endif
+half _ClearCoatSmoothness;
```

Comment:
I think this is fine for now, but maybe we should have a UnityPerMaterial CBuffer specific to ComplexLit. That might require some other changes as well given the fallback mechanism in ComplexLit.